Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx

```diff
@@ -1,16 +1,60 @@
+import {css, StyleSheet} from 'aphrodite';
 import * as React from 'react';
+import {BurgerButton, HBoxLayout, Padding, VBoxLayout} from '../..';
+import {NotificationButton} from './notification_button';
 
 export interface Properties {}
 
-export interface State {}
+export interface State {
+}
 
 /** Displays the main dashboard. */
 export class DashboardPage extends React.Component<Properties, State> {
-  constructor(properties: Properties) {
-    super(properties);
+  constructor(props: Properties) {
+    super(props);
   }
 
   public render(): JSX.Element {
-    return <div>Dashboard</div>;
+    return (
+      <VBoxLayout width='100%' height='100%'>
+        <HBoxLayout width='100%' height='75px'
+          className={css(DashboardPage.STYLE.header)}>
+          <HBoxLayout width='136px' height='75px'>
+            <Padding size='18px'/>
+            <VBoxLayout height='75px'>
+              <Padding/>
+              <VBoxLayout height='30px'>
+                <Padding size='7.5px'/>
+                <BurgerButton width='20px' height='15px'
+                  color='#4B23A0' highlightColor='#4B23A0'/>
```

Comment:
Use the color provided in the specification. If you decide to change the color with Darryl, you must ensure that the change is reflected in the specification before a pull request is submitted, otherwise it will get rejected.